bitwarden_client.go

```diff
@@ -3,7 +3,7 @@ package sdk
 import (
 	"encoding/json"
 
-	"github.com/bitwarden/sdk-go/internal/cinterface"
+	"github.com/mattscamp/sdk-go/internal/cinterface"
```

Comment:
You left in the module rewrites.

Reply:
Oof. Thought I pushed this to a different branch. Will fix ASAP.